features/scv/jceks-impl/src/main/java/org/opennms/features/scv/jceks/JCEKSSecureCredentialsVault.java

```diff
@@ -86,11 +87,15 @@ public JCEKSSecureCredentialsVault(String keystoreFile, String password, byte[]
     }
 
     public JCEKSSecureCredentialsVault(String keystoreFile, String password, byte[] salt, int iterationCount, int keyLength) {
-        m_password = Objects.requireNonNull(password).toCharArray();
         m_salt = Objects.requireNonNull(salt);
         m_iterationCount = iterationCount;
         m_keyLength = keyLength;
         m_keystoreFile = new File(keystoreFile);
+        String keyFromProperties = System.getProperty(SCV_KEY_PROPERTY);
```

Comment:
What's the purpose of this override? This logic should probably be in the application context instead.